src/test/java/school/hei/haapi/integration/MonitorIT.java

```diff
@@ -0,0 +1,186 @@
+package school.hei.haapi.integration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+import static school.hei.haapi.endpoint.rest.model.EnableStatus.ENABLED;
+import static school.hei.haapi.endpoint.rest.model.ProfessionalExperienceFileTypeEnum.WORKER_STUDENT;
+import static school.hei.haapi.endpoint.rest.model.Sex.F;
+import static school.hei.haapi.endpoint.rest.model.Sex.M;
+import static school.hei.haapi.endpoint.rest.model.SpecializationField.COMMON_CORE;
+import static school.hei.haapi.endpoint.rest.model.WorkStudyStatus.WORKING;
+import static school.hei.haapi.integration.conf.TestUtils.MANAGER1_TOKEN;
+import static school.hei.haapi.integration.conf.TestUtils.coordinatesWithNullValues;
+import static school.hei.haapi.integration.conf.TestUtils.group1;
+import static school.hei.haapi.integration.conf.TestUtils.group2;
+import static school.hei.haapi.integration.conf.TestUtils.setUpCognito;
+import static school.hei.haapi.integration.conf.TestUtils.setUpEventBridge;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import school.hei.haapi.endpoint.rest.api.UsersApi;
+import school.hei.haapi.endpoint.rest.client.ApiClient;
+import school.hei.haapi.endpoint.rest.client.ApiException;
+import school.hei.haapi.endpoint.rest.model.Coordinates;
+import school.hei.haapi.endpoint.rest.model.CrupdateMonitor;
+import school.hei.haapi.endpoint.rest.model.Monitor;
+import school.hei.haapi.endpoint.rest.model.Student;
+import school.hei.haapi.endpoint.rest.model.UserIdentifier;
+import school.hei.haapi.integration.conf.MockedThirdParties;
+import school.hei.haapi.integration.conf.TestUtils;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
+
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+@Testcontainers
+@ContextConfiguration(initializers = ManagerIT.ContextInitializer.class)
+@AutoConfigureMockMvc
+public class MonitorIT extends MockedThirdParties {
+  @MockBean private EventBridgeClient eventBridgeClientMock;
+
+  @Autowired ObjectMapper objectMapper;
+
+  private static ApiClient anApiClient(String token) {
+    return TestUtils.anApiClient(token, ManagerIT.ContextInitializer.SERVER_PORT);
+  }
+
+  public static Student student1() {
+    Student student = new Student();
```

Comment:
You no need to rewrite this method here, it's already public static in TestUtils if i'm not wrong. Then import statically directly in need

Reply:
same for those method next to this